receiver/pluginreceiver/plugin_test.go

```diff
@@ -492,3 +494,44 @@ func TestCheckParameters(t *testing.T) {
 		})
 	}
 }
+
+func TestCheckExtensions(t *testing.T) {
+	tmpDir := "/tmp/dir"
```

Comment:
Can we use `t.TmpDir` as these tests won't work on Windows. That way the test structure will take care of cleaning it up too.